src/styles/side-text.scss

```diff
@@ -0,0 +1,26 @@
+.side-text {
+  position: absolute;
+
+  font-family: SharpSansNo2Medium, sans-serif;
+  font-size: 14px;
+  line-height: 130%;
+  letter-spacing: 0.05em;
+  color: rgba(52, 79, 16, 0.5);
+  text-transform: uppercase;
+
+  transform: rotate(-90deg);
+  transform-origin: left top;
+  transition: left $transition-duration;
```

Comment:
https://codeburst.io/low-performance-css-transitions-and-some-alternatives-f84ff35bf07